packages/kbn-securitysolution-grouping/src/components/grouping.mock.tsx

```diff
@@ -24,13 +22,13 @@ export const mockGroupingProps = {
       sum_other_doc_count: 0,
       buckets: [
         {
-          key: [rule1Name, rule1Desc],
-          key_as_string: `${rule1Name}|${rule1Desc}`,
+          key: [host1Name],
```

Comment:
I changed this to be a host query since its more realistic for testing, rule name will not have a null group.